api/urls.py

```diff
@@ -17,6 +17,7 @@
     path("plants/<int:pk>", views.PlantRetrieveView.as_view(), name="single_plant"),
     path("plant-parts/", views.PlantPartListView.as_view(), name="plant_part_list"),
     path("other-ingredients/<int:pk>", views.IngredientRetrieveView.as_view(), name="single_ingredient"),
+    path("microorganisms/", views.MicroorganismCreateView.as_view(), name="microorganism_list"),
```

Comment:
Comme pour la PR des plantes, je changerais le nom de la route :

```suggestion
    path("microorganisms/", views.MicroorganismCreateView.as_view(), name="microorganism_create"),
```